src/test/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceServiceSpec.scala

```diff
@@ -143,4 +143,49 @@ class ResourceServiceSpec extends FlatSpec with Matchers with TestSupport with B
     //cleanup
     runAndWait(service.deleteResource(resourceType, resourceName))
   }
+
+  it should "listUserResourceRoles when they have at least one role" in {
+    val ownerRoleName = ResourceRoleName("owner")
+    val resourceType = ResourceType(ResourceTypeName(UUID.randomUUID().toString), Set(ResourceAction("a1"), ResourceAction("a2"), ResourceAction("a3")), Set(ResourceRole(ownerRoleName, Set(ResourceAction("a1"), ResourceAction("a2")))), ownerRoleName)
```

Comment:
Are the `ResourceAction`s important to this test case? I don't see them being referenced other than when creating the resource type here.